### ISSUE TITLE
Question: Translation of prometheus patterns to jmxfetch queries 

### ISSUE THREAD

**Issue by iMajna** [DataDog/jmxfetch#396]
Unfortunately, I cannot find any info about docs about this anywhere and I'm not sure whether it is actually possible when I see implementation but worth of asking. 

What I'm trying is translating following rule: 
```
  - pattern: 'kafka.server<type=app-info><>(commit-id|version): (.+)'
    name: kafka_server_$1_info
    value: 1
    labels:
      $1: "$2"
    help: "Kafka server JMX metric info version and commit-id"
    type: GAUGE
```

What that gives me is following metric: 
```
kafka_server_version_info{version="3.1.0"} 1.0
```

Reason why I'm doing this is because attribute `version` is reported as `double` but in fact it is string.

```
$>info -b kafka.server:type=app-info
#mbean = kafka.server:type=app-info
#class name = org.apache.kafka.common.metrics.JmxReporter$KafkaMbean
# attributes
%0 - commit-id (double, r)
%1 - start-time-ms (double, r)
%2 - version (double, r)
```

So, if I want use something like this: 
```
                    {
                      "include": {
                        "domain": "kafka.server",
                        "bean": "kafka.server:type=app-info",
                        "attribute": {
                          "version": {
                            "alias": "kafka.server.cluster.version",
                            "metric_type": "gauge",
                            "values": {
                              "default": 1
                            }
                          }
                        }
                      }
                    }
```

It will more or less yield with metric `kafka.server.cluster.version` with value `0` but without tag `version=3.1.0` as the one from prometheus rule. 

So, in short, is there a way to somehow send attribute as a label or a tag with its true value along side metric?
